.github/workflows/sync-upstream.yml

```
@@ -0,0 +1,35 @@
+name: Update
+
+on:
+  # Runs at 10:00 UTC every day
+  schedule:
+    - cron:  '0 10 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  sync:
+    name: Sync version with upstream
+    environment: "Candidate Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sync version with upstream
+        uses: snapcrafters/ci/sync-version@main
+        with:
+          token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
+          update-script: |
+            stable_version=$(curl -sL https://www.sublimemerge.com/download | grep -Po '<p class="latest"><i>Version:</i> Build \K[0-9]+')
+            yq -i ".version = \"$stable_version\"" snap/snapcraft.yaml
```

I'd recommend against using `yq` here, as it will reformat a lot of the yaml file.

```suggestion
            sed -i 's/^\(version: \).*$/\1'"$stable_version"'/' snap/snapcraft.yaml
```

---

I saw that thing happen, so, should I replace all the yqs (in other repos) with sed?

---

?